src/Plugin.php

```diff
@@ -90,7 +90,9 @@ public static function init() {
     // Output client-side cookie notice on all pages.
     // Use a slightly higher weight to prevent the bar being output
     // before other footer content.
-    add_action('wp_footer', __CLASS__ . '::wp_footer', 12);
+    if (!defined('CORE_STANDARDS_COOKIE_NOTICE') || CORE_STANDARDS_COOKIE_NOTICE === FALSE) {
```

Comment:
Your code will prevent the cookie notice from rendering if the constant set and `TRUE`.

```suggestion
    if (!defined('CORE_STANDARDS_COOKIE_NOTICE') || CORE_STANDARDS_COOKIE_NOTICE !== FALSE) {
```